e2e/pages/maps/planting.py

```diff
@@ -22,6 +23,13 @@ def __init__(self, page: Page):
         self.canvas = self.page.get_by_test_id("canvas")
         self.undo_button = self.page.get_by_test_id("undo-button")
         self.redo_button = self.page.get_by_test_id("redo-button")
+        # Base layer
+        self.background_select = self.page.get_by_test_id("baseBackgroundSelect")
+        self.background_button = page.get_by_role("button", name="Choose an image")
+        self.rotation_input = self.page.get_by_test_id("rotation-input")
+        self.scale_input = self.page.get_by_test_id("scale-input")
```

Comment:
This is actually totally unrelated to Planting, it should be in a separate class/file?